xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -392,6 +392,22 @@
     },
     "additionalProperties": { "type": [ "null", { "$ref": "Global.String.NotEmpty", "required": true } ] }
   },
+  "Video.Rating": {
+    "type": "object",
+    "properties": {
+      "rating": { "type": "number", "required": true },
+      "votes": { "type": "integer" },
+      "default": { "type": "boolean" }
+    }
+  },
+  "Video.Ratings": {
+    "type": "object",
+    "additionalProperties": { "type": "$ref": "Video.Rating" }
+  },
+  "Video.Ratings.Set": {
+    "type": "object",
+    "additionalProperties": { "type": [ "null", { "$ref": "Video.Rating", "required": true } ] }
```

Comment:
If I didn't something wrong they're not redundant.
Video.Rating should be:
```
{ "rating": 1, "votes": 2 }
```
Video.Ratings should be:
```
{ "imdb": (Video.Rating), "tvdb": (Video.Rating) }
```